tests/integration_tests/ui_tests/ArelleGUITest/ArelleGUITest.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32929.385
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ArelleGUITest", "ArelleGUITest\ArelleGUITest.csproj", "{0CEE6A09-141B-4DAA-9409-4416A50E60F0}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{0CEE6A09-141B-4DAA-9409-4416A50E60F0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
Any way to alias this?

Reply:
It seems like this maybe doesn't need to exist unless we want people to be using multiple projects? If we are going to keep this, it seems specifically catered towards visual studio (since we don't have multiple projects) so we should address the newlines by providing a config file also [here](https://github.com/Arelle/Arelle/pull/570#discussion_r1097676712)